Source/WebCore/editing/cocoa/WebContentReaderCocoa.mm

```diff
@@ -722,15 +722,13 @@ static String typeForAttachmentElement(const String& contentType)
     bool isDirectory = fileType == FileSystem::FileType::Directory;
     String contentType = typeForAttachmentElement(explicitContentType);
     if (contentType.isEmpty()) {
-ALLOW_DEPRECATED_DECLARATIONS_BEGIN
         if (isDirectory)
-            contentType = kUTTypeDirectory;
+            contentType = String(UTTypeDirectory.identifier);
```

Comment:
No need for `String()` here.

Reply:
Thank you for taking the time to review this!  I'm iterating on the patch as I try to work through some adoption issues with the new API. I'll definitely incorporate these changes (and appreciate them!), but I do intend to do a bit more work. I hope I don't want to waste your time. I will do a better job of marking future PR's as DRAFT to help clarify this.

Reply:
(By new API I mean "the not-deprecated API" :-) )

Reply:
No worries!